Hermes/HermesNotification.swift

```diff
@@ -19,13 +20,27 @@ public class HermesNotification: NSObject {
       return attributedText?.string
     }
   }
+  
+  /// Text, but with fancy attributes instead of a regular `String`
   public var attributedText: NSAttributedString?
+  
+  /// The color of the bottom bar on the notification
   public var color: UIColor?
+  
+  /// The image to be displayed along with the notification
   public var image: UIImage?
+  
+  /// The URL to the image
   public var imageURL: NSURL?
   public var tag: String?
```

Comment:
tag? It's not used in the demo. It's completely optional, and is sometimes useful if you have specially designed notifications.